File: PEMROGRAMAN 1/PRAKTIKUM MODUL 2/Bahasa C/PRAK205-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>
#include <math.h>

int main() {
    //test case 1
    float A1, B1, C1, alas1, tinggi1, keliling1, luas1;

    printf("A dan B:\n");
    scanf("%f", &A1);
    scanf("%f", &B1);

    tinggi1 = A1;
    C1 = B1;
    alas1 = sqrt(pow(B1, 2) - pow(A1, 2));
    keliling1 = alas1 + tinggi1 + C1;
    luas1 = 0.5 * alas1 * tinggi1;

    printf("Alas = %.f cm\n", alas1);
    printf("Tinggi = %.f cm\n", tinggi1);
    printf("Keliling = %.f cm\n", keliling1);
    printf("Luas = %.f cm^2\n", luas1);

    printf("\n");
    //test case 2
    float A2, B2, C2, alas2, tinggi2, keliling2, luas2;

    printf("A dan B:\n");
    scanf("%f", &A2);
    scanf("%f", &B2);

    tinggi2 = A2;
    C2 = B2;
    alas2 = sqrt(pow(B2, 2) - pow(A2, 2));
    keliling2 = alas2 + tinggi2 + C2;
    luas2 = 0.5 * alas2 * tinggi2;

    printf("Alas = %.f cm\n", alas2);
    printf("Tinggi = %.f cm\n", tinggi2);
    printf("Keliling = %.f cm\n", keliling2);
    printf("Luas = %.f cm^2\n", luas2);

    return 0;
}

File: PEMROGRAMAN 1/PRAKTIKUM MODUL 2/Bahasa Python/PRAK205-2410817310009-Muhammad_Ghazi_Rakhmadi.py
import math
# test case 1
A, B = map(float, input("A dan B :\n").split())

tinggi = A
sisi_miring = B
alas = math.sqrt(B**2 - A**2)
keliling = alas + tinggi + sisi_miring
luas = 0.5 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2\n")
# test case 2
A = float (input("A dan B :\n"))
B = float (input(""))

tinggi = A
sisi_miring = B
alas = math.sqrt(B**2 - A**2)
keliling = alas + tinggi + sisi_miring
luas = 0.5 * alas * tinggi

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")